src/common/dataModels/tasks.ts

```diff
@@ -51,6 +51,10 @@ export interface IFindInactiveTasksRequest {
   ignoreTypes?: ITaskType[];
 }
 
+export interface IReleaseInactiveQuery {
```

Comment:
```suggestion
export interface IReleaseInactiveQueryParams {
```

Change ref that is in use